src/scripts/serverNotifications.js

```diff
@@ -1,211 +1,216 @@
-define(['connectionManager', 'playbackManager', 'syncPlayManager', 'events', 'inputManager', 'focusManager', 'appRouter'], function (connectionManager, playbackManager, syncPlayManager, events, inputManager, focusManager, appRouter) {
-    'use strict';
+import connectionManager from 'connectionManager';
+import playbackManager from 'playbackManager';
+import syncPlayManager from 'syncPlayManager';
+import events from 'events';
+import inputManager from 'inputManager';
+import focusManager from 'focusManager';
+import appRouter from 'appRouter';
 
-    var serverNotifications = {};
+const serverNotifications = {};
 
-    function notifyApp() {
-        inputManager.notify();
-    }
+function notifyApp() {
+    inputManager.notify();
+}
 
-    function displayMessage(cmd) {
-        var args = cmd.Arguments;
-        if (args.TimeoutMs) {
-            require(['toast'], function (toast) {
-                toast({ title: args.Header, text: args.Text });
-            });
-        } else {
-            require(['alert'], function (alert) {
-                alert.default({ title: args.Header, text: args.Text });
-            });
-        }
+function displayMessage(cmd) {
+    const args = cmd.Arguments;
+    if (args.TimeoutMs) {
+        require(['toast'], function (toast) {
```

Comment:
```suggestion
        import('toast').then(({default: toast}) => {
```

Reply:
>  are we preferring the latter from now on?

I think it needs to be discussed with the rest of @jellyfin/web. Personally, I like consistency, so I'd say "always use parentheses". That way there's really no guessing involved.

I plan to add an ESLint rule for whatever we end up on, so please give your opinions :)

Reply:
I agree, I think always using parentheses looks better.